activestorage/app/controllers/active_storage/direct_uploads_controller.rb

```diff
@@ -5,13 +5,17 @@
 # the blob that was created up front.
 class ActiveStorage::DirectUploadsController < ActiveStorage::BaseController
   def create
-    blob = ActiveStorage::Blob.create_before_direct_upload!(**blob_args)
+    blob = ActiveStorage::Blob.create_before_direct_upload!(**blob_args, metadata: { "custom" => meta_args[:metadata] })
```

Comment:
Why we need to push the metadata arguments inside the custom here? If we are pushing all the metadata inside custom we don't need to do it in the controller